docs/source/installation/troubleshooting.rst

```diff
@@ -173,7 +175,7 @@ As a last check, you can run
 
   dvisvgm -V1
 
-while still having ``LIBGS`` set to the correct path, of course. If ``dvisvgm``
+(while still having ``LIBGS`` set to the correct path, of course.) If ``dvisvgm``
```

Comment:
Not sure I agree with the parentheses here, but that is more of a personal remark. No change required.